### PR TITLE
Primitive age questbook changes Pt.1

### PR DESCRIPTION
**Pull request by kovreee**
## What
Changes the 0 - Primitive age quest chapter, adding/removing some little things. this is the first part, as I am trying to figure out how all of this works.

## Implementation Details
-Changes the rearrangements of the quests, including dependecies
-Removed some descriptions from some quests that are unnesessary
-Added some descriptions to some quests that will help beginner players
-Didn't delete any quests

## Outcome
Changes the questline in 0 - Primitive age

## Potential Compatibility Issues
Only if I failed to commit/merge/whatever, I might have edited the older version of the book (but iirc marie's changes weren't big, so those could just be re-added)

### CODE REVIEW

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Welcome to §9§9§9§b§b§b§b§b§b§l§l§l§l§l§lSupersymmetry§r§r§r§r§r§r! \n\nYou have just landed on an unfamiliar planet. This book will teach you how to rebuild civilization from scratch in only §k4200§r pages.\n\nStart by getting a rock§r. This quest book is interactive.\n\nWhen you pick up a rock from the ground, press the §l§l§l§l§l§l§a§a§a§a§a§aSUBMIT§r button to complete the quest and continue.\n\nIf you lose this book, you can check the quests by pressing the §2§aGRAVE (`) §rkey.\n\nPages that have a normal icon aren\u0027t required for progression. \nPages that have a framed icon, like this one are §crequired §rand contain critical information for your survival.\n\nYou have been given a §6home block§r. It allows you to set your spawn if you ever die. You cannot skip the night, as beds are not available before EV.\n\n",
```
Comment (spacesailormarie) [inline config/betterquesting/DefaultQuests.json:10]: this is unnecessary

File: config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -6608,7 +6618,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Convert 1 log into 4 §6planks§r. \n\nYou can then craft those planks into a §acrafting table§f.\n\n----------------------------------------\n§3WELCOME TO SUPERSYMMETRY!\n§r----------------------------------------\n- Explore your surroundings\n§r- Find or build shelter\n- Prepare for the winter\n- Acquire basic resources\n- Try not to die\n------------------------------------",
+          "desc:8": "Make a §6Crafting Table§f\n\n----------------------------------------\n§3WELCOME TO SUPERSYMMETRY!\n§r----------------------------------------\n- Explore your surroundings\n§r- Find or build shelter\n- Prepare for the winter\n- Acquire basic resources\n- Try not to die\n------------------------------------",
```
Comment (spacesailormarie) [inline config/betterquesting/DefaultQuests.json:6621]: why

File: config/betterquesting/DefaultQuests.json [hunk 3]
```diff
@@ -9278,7 +9284,7 @@
...
+          "desc:8": "§6Pig Iron§r is the product of smelting iron ore (such as §6Banded Iron§r and §6Magnetite§o§r) with a high-carbon fuel like §6Coal§r, Charcoal, Anthracite, or Coal Coke in a Blast Furnace. §6Limonite §rcan also be used as an ore, but it has to be smelted in a normal Furnace first, or dried by using a Dryer.\n\nThe next step is to make §7§f§7§7§r§7Wrought Iron§ §f out of this Pig Iron, which will be explained in the next quest.\n\nExtracting usable metal from iron ores requires kilns or furnaces (such as a §cBlast Furnace§r) capable of reaching 1,500 °C (1,773K) or higher, about 500 °C higher than that required to smelt copper. \n\n§\n§cIt is recommended to use dusts instead of ores while smelting Iron. If you can\u0027t macerate ores into dusts, look into the Grindstone and Macerator.",
```
Comment (spacesailormarie) [inline config/betterquesting/DefaultQuests.json:9287]: player cant make macerator yet in primitive age

File: config/betterquesting/DefaultQuests.json [hunk 4]
```diff
@@ -90,7 +90,7 @@
...
+          "desc:8": "It is probably impossible to try to play GregTech without using JEI. So here are some tips on how to use JEI to its full potential.\n\nFirst of all, you can press §6R §ron an item in the questbook and in JEI to see the item\u0027s crafting recipe, or §6U §rto see its uses.\n\n§eBookmark items regularly§r (default key §aA§r) so you don\u0027t have to type the names over and over.\n\nMost of the items in this pack have §emultiple recipes§r! Don\u0027t stop at the first recipe and think that item is impossible.\nSometimes a material is only available from §etiny dusts§r, sometimes it\u0027s available from a §efluid solution§r. It takes some practice. Oil processing and nuclear chemistry are pretty bad about this, §6§6ESPECIALLY§f in this pack.\n\n\nYou can §epress @§r to see only items from that mod. For example, if you search for §c@gregtech§r, you will only see items from GregTech.\n\nYou can §epress $ §rto see only items from a specific entry in the Ore Dictionary. For example, if you search for §c$logWood§r, you will only see wooden logs. This feature will be really useful for certain recipies!",
```
Comment (spacesailormarie) [inline config/betterquesting/DefaultQuests.json:93]: nah

**Review by spacesailormarie — CHANGES_REQUESTED** [review]
things i commented on shouldnt change really, i have to look at the positions to see if they look any better too

File: config/betterquesting/DefaultQuests.json (hunk 2 above)
Comment (kovreee) [reply config/betterquesting/DefaultQuests.json:6621]: I think if you are playing susy you know how to make a crafting table but my text could be changed for smth more poetic

File: config/betterquesting/DefaultQuests.json (hunk 3 above)
Comment (kovreee) [reply config/betterquesting/DefaultQuests.json:9287]: we could point that out

File: config/betterquesting/DefaultQuests.json (hunk 4 above)
Comment (kovreee) [reply config/betterquesting/DefaultQuests.json:93]: can I at least remove 100 IQ comment it's stupid

Comment (spacesailormarie) [reply config/betterquesting/DefaultQuests.json:93]: no its funny

Comment (kovreee) [reply config/betterquesting/DefaultQuests.json:93]: true

**Pull request merged**